INSTALL.md

```diff
@@ -2,52 +2,38 @@
 
 ## Rust
 
-The Rust Programming language and `cargo` are required to build the Hash compiler. The easiest way to install Rust is using [rustup](https://rustup.rs/).
+The Rust Programming language and `cargo` are required to build the Hash
+compiler. The easiest way to install Rust is using [rustup](https://rustup.rs/).
 
 ## LLVM
 
-One of the Hash compiler code generation backends is LLVM, so you will need to install LLVM on your system.
-Currently, the compiler used LLVM 15.0.x, which can be installed using the following instructions 
-(specific to each OS):
-
-### MacOS
-
-To install on MacOS, the simplest way to install it is using [Homebrew](https://brew.sh/):
-
-```bash
-brew install llvm@15
-``` 
-**Note**: If you are unsure of where LLVM has been installed, you can query 
-the installation path using `brew --prefix llvm@15`.
-
-
-### Windows
-
-To install on Windows, the simplest way to install it is using [Chocolatey](https://chocolatey.org/):
-
-```pwsh
-choco install llvm --version 15.0.0
```

Comment:
Don't remove this section since the build process will be slightly different, and adding things to the environment path is also different. It should indeed however be 15.0.6 here.

Reply:
Yes I was just about to add it back, I was just wondering about the versions